lib/ret_web/controllers/api/v1/media_controller.ex

```diff
@@ -11,20 +13,31 @@ defmodule RetWeb.Api.V1.MediaController do
   end
 
   def create(conn, %{
-        "media" =>
-          %Plug.Upload{filename: filename, content_type: "application/octet-stream"} = upload
+        "media" => %Plug.Upload{filename: filename, content_type: "application/octet-stream"} = upload,
+        "with_promotion_token" => "true"
```

Comment:
github swallowed my comment -- perhaps `promotion_mode: [always|token]` (leaving option open for `never` eventually?)